components/Card.js

```diff
@@ -1,73 +1,145 @@
 import '../styles/card.sass'
 import HeaderWatermark from '../public/images/card-header-watermark.svg'
-import EthicalBrand from '../public/images/ethical-brand.svg'
+import EthicalBrandLogo from '../public/images/ethical-brand.svg'
 import FbLogo from '../public/images/fb-logo.svg'
 import UserProfile from '../public/images/user-profile.svg'
+import { useState } from 'react'
 
 const Card = ({ className }) => {
   const userName = 'Adrian G.'
-  const connectionStatus = true
+  const userProfile = UserProfile
```

Comment:
I'm just curious, any specific reason for this?

Reply:
Just thought if you're getting the userProfileImage from facebook api you could set it to that variable and it should work.